airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -371,4 +371,11 @@ def fields(self, **kwargs) -> List[str]:
 
         schema = ResourceSchemaLoader(package_name_from_class(self.__class__)).get_schema("ads_insights")
         self._fields = list(schema.get("properties", {}).keys())
+
+        # Having this field in syncs seem to have caused data inaccuracy where fields like `spend` had the wrong values
+        try:
+            self._fields.remove("wish_bid")
+        except ValueError:
+            pass
```

Comment:
Would it have any value in adding logging, even if it is a temp patch?

```

logger.info(f"Ignoring wish_bid because of error generating report using the async API .")
```

Reply:
The log `logger.info(f"Ignoring wish_bid because of error generating report using the async API .")` makes sense in both the case of an exception or not. I don't mind adding it. I don't see a special case in the except clause. Let me know if you see one

Reply:
You are right. It makes sense for both cases; I think it is nice to have it for tracking purposes.

Reply:
Yeah, I was thinking that actually later than a comment would be enough to document the fix.